### PR TITLE
add fiff_readout_eventlist.m; extract events from raw file

### PR DESCRIPTION
**Pull request by zuxfoucault**
A simplified matlab script to extract events from raw file based on find_events function in mne-python/mne/event.py

### CODE REVIEW

File: matlab/fiff_readout_eventlist.m [hunk 1]
```diff
@@ -0,0 +1,98 @@
+function eventlist = fiff_readout_eventlist(fname, stim_chennal, consecutive, output)
```
Comment (agramfort) [inline matlab/fiff_readout_eventlist.m]: call function mne_find_events to be consistent.

stim_chennal -> stim_channel

File: matlab/fiff_readout_eventlist.m [hunk 2]
```diff
@@ -0,0 +1,98 @@
...
+%	Find event from raw file
+%	
+%	Parameters
+%	----------
+%	fname : String, .fiff raw data file name
+%	stim_chennal : the channel that record event
+%   consecutive : bool | 'increasing'
```
Comment (agramfort) [inline matlab/fiff_readout_eventlist.m]: please fix indentation

Comment (agramfort) [reply matlab/fiff_readout_eventlist.m]: but use the way the doc formating is done in matlab see eg

https://github.com/mne-tools/mne-matlab/blob/master/matlab/mne_write_cov.m

File: matlab/mne_find_events.m [hunk 3]
```diff
@@ -0,0 +1,97 @@
+function [eventlist] = mne_find_events(fname, stim_channel, consecutive, output)
+%   
+%   
+%   [eventlist] = mne_find_events(fname, stim_channel, consecutive, output)
+%   
+%   Find event from raw file
+%   
+%   fname        - .fiff raw data file name
+%   stim_channel - the channel that record event
```
Comment (agramfort) [inline matlab/mne_find_events.m]: as a string on integer index?

**Comment by zuxfoucault** [comment]
Thanks for the guide. I made some modification accordingly.

**Comment by agramfort** [comment]
thx @zuxfoucault

**Pull request merged**